source/examples/rapids-sagemaker-hpo/Dockerfile

```diff
@@ -1,27 +0,0 @@
-ARG RAPIDS_IMAGE
```

Comment:
There is code in the notebook that writes this... it's not necessary to have a copy checked into the repo too.